contrib/vagrant/start.sh

```diff
@@ -322,9 +322,12 @@ function write_cilium_cfg() {
         cilium_options+=" --k8s-kubeconfig-path /var/lib/cilium/cilium.kubeconfig"
         cilium_options+=" --kvstore etcd"
         cilium_options+=" --kvstore-opt etcd.config=/var/lib/cilium/etcd-config.yml"
+        cilium_options+=" --ipam=cluster-pool"
```

Comment:
cluster pool is default ipam mode now, as this is explicitly added for cilium_options, do you want to add the same for `cilium_operator_options` ?

Reply:
oh you are right, I just wanted to make it explicit. I'll remove it